TutorialApplication/OrganizationInfoViewController.swift

```diff
@@ -14,6 +14,87 @@ import UIKit
     
     static let kEmployeesOrderHasChanged = "EmployeesOrderHasChanged"
     
+    static let kOrganizationHasChanged = "OrganizationHasChanged"
+    
+    @IBAction func fetchOrganizations()
+    {
+        RequestManager.fetchOrganizations { (response) in
+        
+        var newOrganization:FFOrganization
+        
+        var newEmployee:FFEmployee
+            
+        let countOfOrganizations = DatabaseController.requestResults(for: nil, sortDescriptors: nil, entity: "FFOrganization")
+        print(countOfOrganizations?.count as Any)
+        
+        if (countOfOrganizations?.count)! <= 1
+        {
+        let organizations = response["organizations"]
```

Comment:
if let organizations = response["organizations"] as? [[String : AnyObject]]... and use this format everywhere else